styleguide/source/assets/js/jump-nav.js

```diff
@@ -9,4 +10,44 @@ $(document).ready(function() {
     $(this).parents('#jump-nav').toggleClass('is-closed');
   });
 
-});
+  // sticky jumpnav
+  var StickyElement = function(node){
+    var doc = $(document),
+        fixed = false,
+        anchor = node.find('.jumpnav-anchor'),
+        body = doc.find('.article-body'),
+        content = node.find('.jumpnav-content');
+
+    var onScroll = function(e){
+      var docTop = doc.scrollTop(),
+          anchorTop = anchor.offset().top,
+          stickyBottom = content.offset().top + content.height(),
+          bodyBottom = body.offset().top + body.height();
+
+      if(docTop > anchorTop){
+        if(!fixed){
+          anchor.height(content.outerHeight());
+          content.addClass('fixed');
+          fixed = true;
+        }
+      }  else   {
+        if(fixed){
+          anchor.height(0);
+          content.removeClass('fixed');
+          fixed = false;
+        }
+      }
+
+      if(stickyBottom > bodyBottom){
+        content.hide();
+      } else {
+        content.fadeIn(250);
+      }
+    };
+
+    $(window).on('scroll', onScroll);
+  };
+
+var demo = new StickyElement($('#sticky-jumpnav'));
```

Comment:
@martensc 

yah I left a note about the is-active bit in the description. We (lauren and I) couldn't get it to work with waypoints and that was our best bet. The live site uses some data urls to do it. I think that will need to get resolved in a future ticket.